commit 545d171c1bb21be2d374ccdf1ce8f98d10f1cc0b
Author: Dan Williams <dan.j.williams@intel.com>
Date:   Fri Feb 24 14:55:45 2017 -0800

    mm, devm_memremap_pages: hold device_hotplug lock over mem_hotplug_{begin, done}
    
    commit b5d24fda9c3dce51fcb4eee459550a458eaaf1e2 upstream.
    
    The mem_hotplug_{begin,done} lock coordinates with {get,put}_online_mems()
    to hold off "readers" of the current state of memory from new hotplug
    actions.  mem_hotplug_begin() expects exclusive access, via the
    device_hotplug lock, to set mem_hotplug.active_writer.  Calling
    mem_hotplug_begin() without locking device_hotplug can lead to
    corrupting mem_hotplug.refcount and missed wakeups / soft lockups.
    
    [dan.j.williams@intel.com: v2]
      Link: http://lkml.kernel.org/r/148728203365.38457.17804568297887708345.stgit@dwillia2-desk3.amr.corp.intel.com
    Link: http://lkml.kernel.org/r/148693885680.16345.17802627926777862337.stgit@dwillia2-desk3.amr.corp.intel.com
    Fixes: f931ab479dd2 ("mm: fix devm_memremap_pages crash, use mem_hotplug_{begin, done}")
    Signed-off-by: Dan Williams <dan.j.williams@intel.com>
    Reported-by: Ben Hutchings <ben@decadent.org.uk>
    Cc: Michal Hocko <mhocko@suse.com>
    Cc: Toshi Kani <toshi.kani@hpe.com>
    Cc: Vlastimil Babka <vbabka@suse.cz>
    Cc: Logan Gunthorpe <logang@deltatee.com>
    Cc: Masayoshi Mizuma <m.mizuma@jp.fujitsu.com>
    Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
    Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1250_545d171c1bb21be2d374ccdf1ce8f98d10f1cc0b.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1250_545d171c1bb21be2d374ccdf1ce8f98d10f1cc0b.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
